### ISSUE TITLE
instanceof parsed incorrectly in assertion

### ISSUE THREAD

**Issue by felixhao28** [atom/language-java#187]
### Description

![image](https://user-images.githubusercontent.com/2417391/50828992-fabc6080-137d-11e9-8fdc-67dd932bcdcf.png)

```java
class A {
    void f() {
        A a;
        if (a instanceof A) {
            // ^ keyword.operator.instanceof.java
        }
        assert a instanceof A;
        //         ^ storage.type.java
    }
}
```

"assert" does not work well with "instanceof" while "if" works.

**Comment by sadikovi** [comment]
Yes, I will have a look.

**Comment by sadikovi** [comment]
Seems it works just fine in master. I opened a PR to add test.

**Comment by felixhao28** [comment]
I checked again with the newly pulled syntax file and it indeed works fine. The regression I saw was caused by my attempted fix on #174. Sorry for the false alarm.
